backend_tsc/src/db_conn.ts

```diff
@@ -0,0 +1,15 @@
+import * as mongoose from 'mongoose';
+
+export class Db_conn{
+    public static async init(conn_string:string,caller?:string){
+        return mongoose.connect(conn_string)
+            .then(() => console.log(`${caller ??"Unknown Caller"} connected to db @ ${conn_string}`))
+            .catch((err) => {
+                console.log(`${caller ?? "Unknown Caller"} FAILED connected to db @ ${conn_string}: ${err}`)
+            });
+    }
+
+    public static async db_close(){
+        return mongoose.connection.close();//did not force... but I could have I guess
```

Comment:
What is the difference between force closing and the close you used? If there are no negatives to force closing and it ensures all operations are complete maybe we should force?

Reply:
I'll look into it. If I remember correctly the tutorial I was following used `forceClose()` but I didn't think it was necessary.